### ISSUE TITLE
Error Call undefined method TCPDF::SetAllowLocalFiles()

### ISSUE THREAD

**Issue by SilvaFernando** [yllen/pdf#39]
Hi, i'm using version 2.0.1, when i try print pdf this error occurred:
![image](https://user-images.githubusercontent.com/53239752/216142355-323234bb-d002-4c7a-8977-3b3e302dd747.png)

I'm using php 7.4 and GLPI 9.5.3, can you help-me, please?

**Comment by sutyagin81** [comment]
Hello. Installed GLPI 10.0.6 plugin does not work. The plugin is installed but it is not displayed on the tab for printing. Help me please.

**Comment by SilvaFernando** [comment]
@yllen, yes glpi/vendor/tecnickcom/tcpdf/tcpdf.php is on my environment:
![image](https://user-images.githubusercontent.com/53239752/220964033-84948450-c261-41fb-b928-076e95f74f0a.png)

**Comment by yllen** [comment]
the owner of the tcpdf directory is not correct.
Only files directory must have apache as owner

**Comment by SilvaFernando** [comment]
Hi @yllen.

What owner must on this folder?

**Comment by yllen** [comment]
Same owner as other files in GLPI directory

**Comment by SilvaFernando** [comment]
Hi @yllen.

Owner GLPI directory are same apache too.

**Comment by SilvaFernando** [comment]
Hi @yllen.

I will verify, what version do you use? Can you send me, please?

BR.
Fernando Dias

**Comment by yllen** [comment]
I have the lastest version of 9.5 with tcpdf version 6.4.1

**Comment by yllen** [comment]
@sutyagin81 had you give right in your profile?
